30_differential_abundance.Rmd

```diff
@@ -119,11 +119,13 @@ tse <- subsetByPrevalentTaxa(tse, detection = 0, prevalence = 0.1)
 
 ### ALDEx2
 
-In this section, we will show how to perform a simple ALDEx2 analysis. If you wanted to pick a single method, this method could be recommended to use.
+In this section, we will show how to perform a simple ALDEx2 analysis. 
+If you wanted to pick a single method, this method could be recommended to use.
 According to the developers experience, it tends to identify the common
 features identified by other methods. This statement is in line with a recent
-independent evaluation by @nearingMicrobiomeDifferentialAbundance2022.  Please
-also have a look at the more extensive [vignette](https://bioconductor.org/packages/release/bioc/vignettes/ALDEx2/inst/doc/ALDEx2_vignette.html) 
+independent evaluation by @nearingMicrobiomeDifferentialAbundance2022.  
```

Comment:
In general we would recommend using LastnameYear i.e.  Nearing2022 for the reference keys. You could consider changing it here and elsewhere where relevant but no worries if it feels like too much for now.